SoftwareGuide/Latex/Introduction/Installation.tex

```diff
@@ -84,14 +84,14 @@ \chapter{Configuring and Building ITK}
 \begin{table}[h]
 \begin{center}
 \resizebox{\textwidth}{!}{
-\begin{tabular}{c c c c c c c c c c c c c c c c c}
+\begin{tabular}{c c c c c c c c c c c c}
 \toprule
-Compiler & 2011 & 2012 & 2013 & 2014 & 2015 & 2016 & 2017 & 2018 & 2019 & 2020 & 2021 & 2022 & 2023 & 2024 & 2025 & 2026 \\
+Compiler & 2016 & 2017 & 2018 & 2019 & 2020 & 2021 & 2022 & 2023 & 2024 & 2025 & 2026 \\
 \midrule
-Visual Studio 10 &  \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{yellow} & \cellcolor{red} & \cellcolor{red} & \cellcolor{red} & & \\
-GCC 4.2 & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{yellow} & \cellcolor{red} & \cellcolor{red} & \cellcolor{red} & \cellcolor{red} & \cellcolor{red} & & & & & \\
-GCC 4.4 & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{yellow} & \cellcolor{red} & \cellcolor{red} & \cellcolor{red} & \cellcolor{red} & \cellcolor{red} & & \\
-GCC 4.9 & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{yellow} & \cellcolor{red} \\
+Visual Studio 10 & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{itkTableCellGreen} & \cellcolor{yellow} & \cellcolor{red} & \cellcolor{red} & \cellcolor{red} & & \\
```

Comment:
How does this table look like when rendered? It is hard to compile LaTeX in my head 😄 

Why is `clang` left out? And only VS version 10 mentioned? We should better update this table or remove it completely.

GCC is at version 10.1 right now. And this table will only grow more outdated with time. I vote we remove it. It was important 15 years ago, when compilers did not adhere well to the C++ standard. Now far less so.

Reply:
To see how the existing table looks like, we'd need to visit the current PDF version. Also, if I remember correctly, one of the CIs that Matt set up was building the PDF, but I was unable to retrieve it for the PRs I made yesterday.

As for the content itself, `clang` was already not part of the table before this commit. Not sure why.I do not use `clang` so I would not be able to add the appropriate information, sorry.

I agree the table is hard to maintain, but OTOH the community should know which compilers are supported. Best would be to automatically extract that from CDash (and put it may be elsewhere), but in absence of that mechanism, this table may do the job. As for the specific compiler versions out there now and their deprecation cycles, I do not follow those news, sorry.

Reply:
Rendered:

![image](https://user-images.githubusercontent.com/25432/81962130-4cfc3a00-95e1-11ea-9fa8-e129df6807bb.png)

Reply:
👍  Thanks for putting in place the GitHub Actions workflow Matt. Is there some link where the PDFs are accessible? I have not seen any URL in the logs.

Reply:
The table is outdated, and the years are difficult to estimate. Compiler releases occur much more quickly than they used to, and their support life is often limited.

We could remove the dates here and add the list compilers know to be supported, i.e.

- GCC 4.8 and newer.
- Visual Studio 2015, 2017, 2019, toolsets 14.x, 15.0, 16.0
- Apple Clang 7.0 and newer.
- Clang 6 and newer.

Reply:
I agree, but if a list of compilers is given, it must clearly state that the list of compilers is only for the date at which the document was generated.

Reply:
I'm fine with whatever you think fits best. If Matt's proposal is the way to go, I can remove the table, shorten and reformat the list to the one Matt proposed, and push force a new version.

Reply:
I agree with Matt's proposal. Thanks for updating this Jon!

Reply:
Changed in a5f9673.